src/v/cloud_storage/remote_partition.cc

```diff
@@ -1316,7 +1315,8 @@ ss::future<> finalize_background(remote& api, finalize_data data) {
         // Remote and local state is in sync, no action required.
         vlog(
           cst_log.debug,
-          "[{}] Remote manifest is in sync with local state during finalize "
+          "[{}] Remote manifest is in sync with local state during "
+          "finalize "
           "(insync_offset={})",
```

Comment:
nit: it's a shame clang-format does not this automatically
if you happen to push other commits:

```suggestion
          "finalize (insync_offset={})",
```

Reply:
will add this in the next PR